src/reactions-bot/src/main/kotlin/org/icpclive/reacbot/Storage.kt

```diff
@@ -47,21 +47,21 @@ class Storage {
                 )
             val voted = Votes.selectAll()
                 .where { Votes.chatId eq chatId }
-                .map { Vote.wrapRow(it).id.value }
+                .map { Vote.wrapRow(it).runId }
```

Comment:
It looks like there was some bug here. vote ids and run ids was messed up. But I don't really sure what's going on.

Reply:
May be
I want to rewrite this code